admins.md

```diff
@@ -8,7 +8,7 @@ All admins of this community are fulfilling that duty in a volunteer capacity. P
 | :--: | :--- | :---------- | :--------: |
 |![Photo of Steve Holsinger](https://ca.slack-edge.com/TAU70QQBY-UAV0HQQJK-9d8b4e70c62e-72)|[Steve Holsinger](https://sfcc-unofficial.slack.com/team/UAV0HQQJK)|Co-founder of the 2014/15 original version of the SFCC B2C Slack which eventually was accidentally closed to the public in May of 2018. In addition to volunteering in this community, Steve is also a Technical Practice Director of Salesforce Commerce @ XCentium. Steve has 10+ years of experience in SFCC B2C/Demandware development & architecture.| [sholsinger](https://github.com/sholsinger) |
 |![Avatar of Matt Rose](https://ca.slack-edge.com/TAU70QQBY-UAT3HAL92-gfe2e106fbcb-72)|[Matt Rose](https://sfcc-unofficial.slack.com/team/UAT3HAL92)| Matt has 10+ years of experience in SFCC B2C/Demandware across various technical roles related to design, development and architecture. Matt created the current SFCC Unofficial Slack Community after the previously mentioned original community was closed. Matt is currently working as an independent contractor after working at Salesforce and various partner agencies.  | [matthewrose](https://github.com/matthewrose) |
-|![Photo of Oleg Sapishchuk](https://ca.slack-edge.com/TAU70QQBY-UASPSP28H-776050e8d86e-72)|[Oleg Sapishchuk](https://sfcc-unofficial.slack.com/team/UASPSP28H)| | |
+|![Photo of Oleg Sapishchuk](https://ca.slack-edge.com/TAU70QQBY-UASPSP28H-776050e8d86e-72)|[Oleg Sapishchuk](https://sfcc-unofficial.slack.com/team/UASPSP28H)| Oleg's overall goal is to become a thought leader that leads and teaches by example. Oleg want's continuously to build products that will bring positive impacts to people. Working as a Technical Architect at Salesforce, Oleg promotes community recognition and support from Salesforce. In addition to mentioned above, Oleg is Co-founder of the Unofficial Salesforce Commerce Cloud Podcast, an initiative that is running under the "Unofficial" Community umbrella.| [Gektorian](https://github.com/Gektorian) |
```

Comment:
Small typo - I'd suggest "wants" instead of "want's" @Gektorian

Reply:
Solved :)